jest.config.js

```diff
@@ -0,0 +1,189 @@
+// For a detailed explanation regarding each configuration property, visit:
+// https://jestjs.io/docs/en/configuration.html
+
+module.exports = {
+  // All imported modules in your tests should be mocked automatically
+  // automock: false,
+
+  // Stop running tests after `n` failures
+  // bail: 0,
+
+  // Respect "browser" field in package.json when resolving modules
+  // browser: false,
+
+  // The directory where Jest should store its cached dependency information
+  // cacheDirectory: "/private/var/folders/v2/r_v842115xs2kkpt8hwxdrt80000gn/T/jest_dx",
+
+  // Automatically clear mock calls and instances between every test
+  clearMocks: true,
+
+  // Indicates whether the coverage information should be collected while executing the test
+  collectCoverage: true,
+
+  // An array of glob patterns indicating a set of files for which coverage information should be collected
+  collectCoverageFrom: [
+    // TODO: Once all tests have been added for existing files, default to all JavaScript files
+    // "./**/*.js",
+    './exchanges/Ethfinex.js',
+  ],
+
+  // The directory where Jest should output its coverage files
+  coverageDirectory: 'coverage',
+
+  // An array of regexp pattern strings used to skip coverage collection
+  coveragePathIgnorePatterns: ['/node_modules/', './coverage/', './generateTokensBySymbol.js', './*.config.js'],
+
+  // A list of reporter names that Jest uses when writing coverage reports
+  coverageReporters: ['text', 'lcov'],
+
+  // An object that configures minimum threshold enforcement for coverage results
+  coverageThreshold: {
+    global: {
+      branches: 100,
+      // functions: 90,
```

Comment:
Was having trouble getting this high on the functions since I was mocking the functions that make request calls. Same w.r.t the line coverage.
```
-------------|----------|----------|----------|----------|-------------------|
File         |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
-------------|----------|----------|----------|----------|-------------------|
All files    |    86.84 |      100 |    77.78 |    86.49 |                   |
 Ethfinex.js |    86.84 |      100 |    77.78 |    86.49 |    37,44,48,50,57 |
-------------|----------|----------|----------|----------|-------------------|
```